libs/camera-streamer.sh

```diff
@@ -36,11 +36,24 @@ function run_ayucamstream() {
     rtsp=$(get_param "cam ${cam_sec}" enable_rtsp)
     rtsp_pt=$(get_param "cam ${cam_sec}" rtsp_port)
     cstm="$(get_param "cam ${cam_sec}" custom_flags 2> /dev/null)"
+    noprx="$(get_param "crowsnest" no_proxy 2> /dev/null)"
     ## construct start parameter
     # set http port
     #
     start_param=( --http-port="${pt}" )
 
+    if [[ -n "${noprx}" ]] && [[ "${noprx}" = "true" ]]; then
+        # See https://github.com/mainsail-crew/crowsnest/pull/221#issuecomment-1863555700 for why
+        # we cannot assume the binary has support for `--http-listen`.
+        if $ust_bin --help | grep -q "http-listen"; then
+            start_param+=( --http-listen "0.0.0.0" )
+            log_msg "INFO: Set to 'no_proxy' mode! Using 0.0.0.0 !"
```

Comment:
Small typo. Seems like ustreamer.sh has the same typo.
There shouldn't be a space in front of the `!`